dsc/examples/wmi.dsc.yaml

```diff
@@ -0,0 +1,12 @@
+$schema: https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2024/04/config/document.json
+resources:
+- name: WMI
+  type: Microsoft.Windows/WMI
+  properties:
+    resources:
+      - name: computer system
+        type: root.cimv2/Win32_ComputerSystem
+        properties:
```

Comment:
Looks like we need to modify the schema to make this 'properties' field optional if no are specified.

Reply:
Actually, `properties` is already optional, so removing from example rather than having it empty